### DISCUSSION TITLE
配置了babel-plugin-dynamic-import-node后，react.lazy无同时导致不生效，如何处理？

### DISCUSSION THREAD

**Discussion by wenlei0617** [umijs/umi#12270] [Q&A]
配置了babel-plugin-dynamic-import-node后，react.lazy无同时导致不生效。场景，不期望使用umi自带的打包方式，期望自行处理，当前使用babel-plugin-dynamic-import-node就无法使用react.lazy的分包了。

**Comment by fz6m** [comment]
配置这个试下：

```ts
// .umirc.ts

  routeLoader: { moduleType: 'cjs' }
```

不过这个选项有个 bug （不影响使用），你可以先用。
